CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## Unreleased
+
+* UI Updates to previous and next navigation component on mobile ([PR #1305](https://github.com/alphagov/govuk_publishing_components/pull/1305))
```

Comment:
Nitpick: can this and the PR title be the same please?

Reply:
have made consistent